backend/src/models/note.ts

```diff
@@ -5,10 +5,13 @@ const noteSchema = new Schema(
     userId: { type: Schema.Types.ObjectId, required: true },
     title: { type: String, required: true },
     text: { type: String },
+    deletedAt: { type: Date, default: undefined },
   },
   { timestamps: true }
 );
 
+noteSchema.index({ deletedAt: 1 }, { expireAfterSeconds: 86400 });
```

Comment:
what is `deletedAt: 1`?

Reply:
It means deletedAt field exist. If deletedAt added to any note expireAfterSeconds starts